CRM/MembershipExtras/Hook/Links/RecurringContribution.php

```diff
@@ -72,7 +73,7 @@ private function getRecurringContribution() {
    */
   public function alterLinks() {
     foreach ($this->links as &$actionLink) {
-      if ($actionLink['name'] == 'Cancel') {
+      if ($actionLink['name'] == 'Cancel' && $this->isMembership() && $this->isManualPaymentPlan()) {
```

Comment:
I don't think we need the `isMembership()` check and that `isManualPaymentPlan()` is enough, reason is we don't support payment plans without memberships yet in Membershipextras.

Reply:
Thanks, chief.